Code/Scripts/CMakeLists.txt

```diff
@@ -110,17 +112,29 @@ if(SV_USE_TKCXIMAGE)
 	endif()
 endif()
 
+if (SV_USE_SYSTEM_TCL)
+	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.6" developer_script_string)
+	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.6" developer_script_string)
+endif()
+
 if(NOT SV_USE_SYSTEM_TCL OR SV_INSTALL_SYSTEM_LIBS)
 	#Lib additions
-	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.5" developer_script_string)
-	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.5" developer_script_string)
-	set_env_string_concat(TCL_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tcl8.5" install_script_string)
-	set_env_string_concat(TK_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tk8.5" install_script_string)
+	set_env_string_concat(TCL_LIBRARY "${TCL_LIBRARY_PATH}/tcl8.6" developer_script_string)
+	set_env_string_concat(TK_LIBRARY "${TK_LIBRARY_PATH}/tk8.6" developer_script_string)
+	set_env_string_concat(TCL_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tcl8.6" install_script_string)
+	set_env_string_concat(TK_LIBRARY "${ENV_SV_HOME}/${SV_INSTALL_TCL_LIBRARY_DIR}/tk8.6" install_script_string)
 endif()
 
 #--------
 # Python
 if(SV_USE_PYTHON)
+  if (SV_USE_SYSTEM_PYTHON)
+	   append_env_string_concat(PYTHONPATH "${PYTHON_SITE_PACKAGES}" developer_script_string)
+	if (SV_USE_SYSTEM_VTK)
```

Comment:
Yes, this was set up for the build (not install) structure because we used to not install vtk. We can set it up for the install structure, but we should also install vtk everytime by default as well. I think we decided we were going to do that anyway last week. Anyway, either way should be fine as long as we are consistent between packages and platforms as much as possible.

I agree that VTK_PYTHON_SITE_PACKAGES should need to be specified if SV_USE_VTK and SV_USE_PYTHON.